src/utils/apiAuth.js

```diff
@@ -88,14 +109,18 @@ export const forwardAPIRequest = async (req) => {
     return response
   })
 
-  const { data } = await api({
-    method: req.method,
-    headers,
-    url: `${BONUSCALC_SERVICE_API_URL}/${path?.join('/')}`,
-    params: queryParams,
-    paramsSerializer: paramsSerializer,
-    data: req.body ? req.body : undefined,
-  })
-
-  return data
+  try {
+    const { data } = await api({
+      method: req.method,
+      headers,
+      url: `${BONUSCALC_SERVICE_API_URL}/${path?.join('/')}`,
+      params: queryParams,
+      paramsSerializer: paramsSerializer,
+      data: req.body ? req.body : undefined,
+    })
+
+    return data
+  } catch (error) {
+    throw Error(error)
+  }
```

Comment:
There was a number of issues on the Sentry GitHub repo that talked about async code not being isolated - this is something to watch out for once we deploy it.